conventions/code-reviews.md

```diff
@@ -0,0 +1,74 @@
+2. Catch Bugs:
+    * They catch potential bugs from being integrated into the main code base
+    * They ensure code quality and veracity
+3. Improve Code:
+    * Improve your code base by having another set of eyes on it
+    * Catch security vulnerabilities that you may not have known about
+    * Give more developers the ability to move between projects without the initial shock of not knowing _anything_ about the code
+
+## Process
+
+### Author
+* At Shift3 we have a simple code review process
+
+1. Before you create your Pull Request or tag anyone in anything, double check that you're ready
+
+    Pre-PR checklist:   
+    - [ ] Test the specific code you're submitting
+    - [ ] Use a linter (esLint is probably going to be the standard) to take care of whitespace and styling
+    - [ ] Remove your console. s
+    - [ ] Make sure your code changes are uner 500 lines of code (break it up into more than 1 PR if it exceeds this)
+
+2. Submit your pull request and tag potential reviewers. Keep in mind that each reviewer should: 
+    * Be either on your project or a quick study on the stack your project is in
+    * Not always be the same person/people (mix it up!)
+    * Understand the purpose of the PR
+
+3. Add comments when you create the PR that explain its scope, purpose, and any quirky things that you had to do to the code to make it work. Explain as much as possible in these comments to let the reviewer know about the PR.  
+
+4. If you recieve a blocked review, fix up the issues and push the changes to your branch. Let the reviewer know when you have submitted the fix. Remember, we all make mistakes, and we all have room to grow. This is an opportunity to learn.
+
+### Reviewer
+
+1. Review any PRs you are tagged in immediately if possible
+    - Here are some general guidelines to help you in your review
+        * Use Github's Review dropdown framework and add your comments inline in the code
+
+        * Limit your review time to half an hour (if the PR takes longer than that to review, suggest to the author that they break it up)
+
+        * Pull the code in and run it on your machine (if possible) to test it
+
+        * Don’t block the PR unless there is a legitimate bug
+
+        * Frame your feedback as requests, not commands and be kind but direct (avoid using personal pronouns i.e. “Your code sucks…” “My code worked before you messed it up…”)
+
+        * Provide code examples, references, links, etc when applicable
+
+        * Don’t forget to compliment the sumitter on good stuff, too!
+
+2. Mark your review as one of the following: 
+    `Request Changes`: If there are bugs in the code or something unacceptable being introduced to the code base (swearing in comments, console. s etc.), you should block the PR and leave detailed comments as to why. If you block a PR and the author resubmits changes, get to them as quickly as you can and approve the PR once it's addressed
```

Comment:
> (swearing in comments, console. s etc.)

Another console.logs word that got truncated.

Reply:
And sumitter should be submitter, I'm rubbing off my terrible spelling on @ryekerjh :joy:

Reply:
@mwallert DOH! I pushed up that change.